api/client/crop_model.py

```diff
@@ -186,16 +219,23 @@ def growing_degree_days(self, region_name, base_temperature,
         has data for the time period, then that will be used. If it's
         a district or other region, the underlying data could be from
         one or more weather stations and/or satellite.  To by-pass the
-        search for available series, use compute_gdd() directly.
+        search for available series, use :meth:`~.compute_gdd` directly.
 
         Parameters
         ----------
-        region_name: string
-        base_temperature: number
-        start_date: '%Y-%m-%d' string
-        end_date: '%Y-%m-%d' string
-        min_temporal_coverage: float, optional
-        upper_temperature_cap: float, optional
+        region_name : string
+        base_temperature : number
+        start_date : string
+            YYYY-MM-DD date
+        end_date : string
+            YYYY-MM-DD date
+        min_temporal_coverage : float, optional
+        upper_temperature_cap : number, optional
```

Comment:
I changed this from float to number. Doesn't have to be a float, right? Figured it should be the same type as base_temperature